src/runtime/GlobalObjectBuiltinTypedArray.cpp

```diff
@@ -211,8 +211,13 @@ Value builtinTypedArrayConstructor(ExecutionState& state, Value thisValue, size_
             uint64_t length = inputObj->length(state);
             ASSERT(length >= 0);
             unsigned elementSize = obj->elementSize();
+            uint64_t buffer_size = length * elementSize;
```

Comment:
I am sorry to bother you.
Could you change buffer_size to bufferSize?
Every otherthings are great

Reply:
Of course! I didn't realize that, I've got used to the underscore naming convention! :)